datafusion/sql/src/unparser/dialect.rs

```diff
@@ -290,6 +297,10 @@ impl Dialect for DuckDBDialect {
         CharacterLengthStyle::Length
     }
 
+    fn division_operator(&self) -> BinaryOperator {
+        BinaryOperator::DuckIntegerDivide
```

Comment:
Should this be `DuckDBIntegerDivide`?

Reply:
No, this is a enum variant defined in sql parser crate and the name is `DuckIntegerDivide `: https://docs.rs/sqlparser/latest/sqlparser/ast/enum.BinaryOperator.html#variant.DuckIntegerDivide

Reply:
Ah, thought we were defining it